Mollier/SAM.Core.Mollier.UI/Query/FindPoints.cs

```diff
@@ -0,0 +1,85 @@
+﻿using SAM.Geometry.Planar;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Windows.Forms.DataVisualization.Charting;
+using System;
+using System.Windows.Forms;
+
+namespace SAM.Core.Mollier.UI
+{
+    public static partial class Query
+    {
+        // query -> znajdowanie punktu o percent : FindPoints( IEnumerable<MollierPoint>, MollierControlSettings, ChartDataType, double percent)
+        [Obsolete("To be changed Maciek")] // TODO: Change (MollierProcess)
+        public static void FindPoints(Control control, Chart chart, MollierControlSettings mollierControlSettings, List<UIMollierPoint> mollierPoints)
```

Comment:
Change from Control to MollierControl. Add the keyword "this". Refactor as discussed